xl/formatter.py

```diff
@@ -850,7 +850,7 @@ def format(self, track, parameters):
             :returns: the formatted value
             :rtype: string
         """
-        value = track.get_tag_disk(self.name)
+        value = track.get_tag_raw(self.name)
```

Comment:
I don't believe comments can use raw tag, otherwise things break... I don't quite remember why this distinction exists, but it does exist. I believe it's because comments aren't currently stored in the database.

Reply:
Hm, actually, that might just be covers. Comment isn't marked as use_disk in tags.py.